map/access-load-status/src/main/java/com/esri/samples/access_load_status/AccessLoadStatusSample.java

```diff
@@ -112,10 +117,10 @@ public void start(Stage stage) {
       // add label, text and button to the control panel
       controlsVBox.getChildren().addAll(loadStatusLabel, loadStatusText, reloadMapButton);
 
-      // create ArcGISMap with the imagery basemap
-      map = new ArcGISMap(Basemap.createImagery());
+      // create a map with the imagery standard basemap style
```

Comment:
Maybe it'd be better to make this into 'plain english' by saying 'standard imagery basemap style'.
I might be wrong but possibly the enums are named in a format to have the 'imagery' appear first, followed by a qualifier (imagery, imagery standard, imagery labels, etc.). It's not 100% clear from the JavaDoc though, so I'll leave that to the phrasing specialists ;)